pkg/controllers/user/logging/utils/wrap.go

```diff
@@ -90,7 +87,7 @@ func ToWrapClusterLogging(clusterLogging v3.ClusterLoggingSpec) (*WrapClusterLog
 		ClusterLoggingSpec: clusterLogging,
 	}
 
-	curtg, wes, wsp, wsl, wkf, err := getWrapConfig(clusterLogging.ElasticsearchConfig, clusterLogging.SplunkConfig, clusterLogging.SyslogConfig, clusterLogging.KafkaConfig)
+	curtg, wes, wsp, wsl, wkf, _, err := getWrapConfig(clusterLogging.ElasticsearchConfig, clusterLogging.SplunkConfig, clusterLogging.SyslogConfig, clusterLogging.KafkaConfig, clusterLogging.EmbeddedConfig)
```

Comment:
I'm curious that the newly added return value is never used. Why do we handle EmbeddedConfig seperately in the following lines, can it be done in `getWrapConfig` function and we simply assign `wp.WrapEmbedded` like the other kinds?